src/EFCore.PG/Metadata/Conventions/NpgsqlPostgresModelFinalizingConvention.cs

```diff
@@ -46,6 +69,7 @@ public virtual void ProcessModelFinalizing(IConventionModelBuilder modelBuilder,
         RelationalTypeMapping typeMapping,
         IConventionModelBuilder modelBuilder)
     {
+        // TODO does not work if CREATE EXTENSION was done on a non-default schema.
```

Comment:
TODO

Reply:
We probably have issues in other places around this, and nobody seems to care so far... Opened #3177 to track.